### PR TITLE
Soft 143 charger controller messages

### PR DESCRIPTION
**Pull request by JarvisWeng**

### CODE REVIEW

File: libraries/codegen-tooling/inc/can_msg_defs.h [hunk 1]
```diff
@@ -24,7 +24,6 @@ typedef enum {
 typedef enum {
   SYSTEM_CAN_MESSAGE_BPS_HEARTBEAT = 0,
   SYSTEM_CAN_MESSAGE_SET_RELAY_STATES = 1,
-  SYSTEM_CAN_MESSAGE_GET_RELAY_STATES = 2,
```
Comment (GeraldNDA) [inline libraries/codegen-tooling/inc/can_msg_defs.h]: Purposely got rid of "GET RELAY STATES"?

Comment (JarvisWeng) [reply libraries/codegen-tooling/inc/can_msg_defs.h]: arshan removed it in the video remember?

**Comment by JarvisWeng** [comment]
> Seems good as long as `SYSTEM_CAN_MESSAGE_GET_RELAY_STATES` isn't/won't be used anywhere.

arshan removed it in his video

File: libraries/codegen-tooling/inc/can_msg_defs.h (hunk 1 above)
Comment (GeraldNDA) [reply libraries/codegen-tooling/inc/can_msg_defs.h]: Ah you merged with #54 ic.

**Pull request merged**